src/schema/objects/suffixes.yaml

```diff
@@ -554,6 +554,45 @@ sbref:
   name: Single-band reference image
   description: |
     Single-band reference for one or more multi-band `dwi` images.
+scans:
+  name: Scans file
+  description: |
+    The purpose of this file is to describe timing and other properties of each imaging acquisition
+    sequence (each run file) within one session.
+
+    Each neural recording file SHOULD be described by exactly one row. Some recordings consist of
+    multiple parts, that span several files, for example through echo-, part-, or split- entities.
+    Such recordings MUST be documented with one row per file.
+
+    Relative paths to files should be used under a compulsory filename header.
+
+    If acquisition time is included it should be listed under the acq_time header.
+    Acquisition time refers to when the first data point in each run was acquired.
+    Furthermore, if this header is provided, the acquisition times of all files that belong to a
+    recording MUST be identical.
+
+    Datetime should be expressed as described in Units.
+
+    For anonymization purposes all dates within one subject should be shifted by a randomly chosen
+    (but consistent across all recordings) number of days.
+    This way relative timing would be preserved, but chances of identifying a person based on the
+    date and time of their scan would be decreased.
+    Dates that are shifted for anonymization purposes SHOULD be set to the year 1925 or earlier to
+    clearly distinguish them from unmodified data. Shifting dates is RECOMMENDED, but not required.
```

Comment:
I think the details about anonymization should only be in the "Units" section of the spec, not in the schema entries, see also: #948 #986